jbi/queue.py

```diff
@@ -205,7 +205,7 @@ async def get(self, bug_id: int) -> AsyncIterator[QueueItem]:
     async def get_all(self) -> dict[int, AsyncIterator[QueueItem]]:
         all_items: dict[int, AsyncIterator[QueueItem]] = {}
         for filesystem_object in self.location.iterdir():
-            if filesystem_object.is_dir():
+            if filesystem_object.is_dir() and re.match("\d", filesystem_object.name): # filtering out temp files from checks
```

Comment:
It looks like you didn't cherry-pick the commit because I don't see the unit test from #978

Reply:
Fixed! Will credit you in merge.